### ISSUE TITLE
Nginx egg php version

### ISSUE THREAD

**Issue by NoodlesDev987** [lazybytez/eggs#14]
How can I change php version for your nginx egg?

**Comment by pascal-zarrad** [comment]
Currently there is no straight forward configuration to change the php version.

You must change the version in the "FROM" statement in our Dockerfile to fir the version you need.
Note that this will require you to either have the image installed locally (prevent Pterodactyl from always pulling a fresh one) or push it to some Docker image registry like hub.docker.com.
![image](https://user-images.githubusercontent.com/14060618/114669795-96c63680-9d02-11eb-991a-8b9296a9bb89.png)

**Comment by pascal-zarrad** [comment]
I will close this issue and mark it as won't fix, as the current solution would be to publish multiple Docker images for the NGINX egg. This is something we currently would like to avoid.

**Comment by NoodlesDev987** [comment]
> I will close this issue and mark it as won't fix, as the current solution would be to publish multiple Docker images for the NGINX egg. This is something we currently would like to avoid.

Why you would like to avoid it? There should be more docker images so you can choose (as for discord.js bots are multiple node versions)

**Comment by pascal-zarrad** [comment]
@NoodlesDev987 It is about the maintainance effort that is neccessary to keep those images updated and tested. We might release some in the future, but only after setting up CD to reduce the effort of deploying further images.

**Comment by NoodlesDev987** [comment]
Ok, hope you will release more versions soon.
